keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,464 @@
+    └── removed
+        └── 0003.yaml
+
+$ kepctl remove 0004 --release v1.22
+==> KEP 0004 moved to removed
+
+$tree releases
+releases
+└── v1.22
+    ├── accepted
+    │   └── 0001.yaml
+    ├── at-risk
+    |   └── 0002.yaml
+    ├── exception
+    ├── OWNERS
+    ├── proposed
+    └── removed
+        └── 0003.yaml
+        └── 0004.yaml
+```
+
+
+### Requesting an Exception for a KEP
+
+To request an exception for the release, the responsible SIG will create a new PR that moves the receipt into the `exception` directory. A PR template  in the `kubernetes/enhancement` repo will capture the relevant information for the exception request. The release team will approve the exception reqeust, but will require confirmation from any impacted reviewers on the PR before it is merged.
+
+
+```
+$ kepctl exception request --release v1.22 --sig api-machinery 1240
+==> api-machinery/1240 moved to exception directory for v1.22.
+==> Please open a pull request to complete this request: link_with_template.
+
+```
+
+The exception request pull request template will contain the questionnaire currently used in the exception request process:
+
+```
+Enhancement name:
+Enhancement status (alpha/beta/stable):
+SIG:
+k/enhancements repo issue #:
+PR #’s:
+Additional time needed (in days):
+Reason this enhancement is critical for this milestone:
+Risks from adding code late: (to k8s stability, testing, etc.)
+Risks from cutting enhancement: (partial implementation, critical customer usecase, etc.)
+```
+
+If an exception request is not granted, the receipt is moved back to the `removed` directory. If the exception is granted, it is moved to the `at-risk` directory until the conditions that caused it to be removed are corrected. `kepctl` will provide convenience methods for this as well:
+
+```
+$ kepctl exception accept --release v1.22 --sig api-machinery 1240
+==> api-machinery/1240 moved to at-risk.
+
+
+$ kepctl exception deny --release v1.22 --sig api-machinery 1240
+==> api-machinery/1240 moved to removed
+```
+
+
+### Generating the release manifest
+
+The current enhancement tracking sheet will no longer be used actively with the receipts process. To provide a concise view of the state of the release, the enhancements team will generate a release manifest that summarizes the `accepted`, `at-risk`, and `removed` enhancements. This manifest will be updated throughout the release cycle.
+
+### Test Plan
+
+This KEP entails a process change, and hence the tooling to enable the processes will have necessary tests.
+
+### Graduation Criteria
+
+#### Alpha (v1.21)
+
+- Define the process to be followed by the SIGs when proposing an enhancement for the release
+- Define the process to be followed by the Enhancements subteam every release cycle
+- Minimal tooling to perform the defined process and validation of receipts
+- Training/communicating the community about the changes to the process
+
+#### Alpha -> Beta Graduation (v1.22) (Proposed)
+
```

Comment:
Can we clarify here: does this mean that it's rolling out to authors/sigs?  Because I think we need a bulletpoint covering a few actual authors/sigs opting in to test this before it becomes mandatory to be used?